src/scenes/Root/Header/ProfileToolbar/ProfileToolbar.tsx

```diff
@@ -46,6 +53,11 @@ export const ProfileToolbar: FC = () => {
         <IconButton>
           <NotificationsNone />
         </IconButton>
+        {!isManagerOpen && (
+          <IconButton onClick={() => setIsManagerOpen(true)}>
+            <CloudUpload />
+          </IconButton>
+        )}
```

Comment:
I wonder if it would be better to put this in the context menu to the right. That seems better than adding another icon here and would give one of the existing icons something to do.